administrate/cache.md

```diff
@@ -28,15 +29,15 @@ To enable it, you just have to create a `varnish.vcl` file in the `/clevercloud`
 This file describes how Varnish caches your applications and how it decides to return a cached resource or not.
 
 {{< alert "warning" >}}
-<p>The <code>vcl 4.0;</code> and backend section of the `varnish.vcl` configuration file are not necessary as they are already handled by Clever Cloud.</p>
+<p>The <code>vcl 4.1;</code> and backend section of the `varnish.vcl` configuration file are not necessary as they are already handled by Clever Cloud.</p>
 <p>If you have a PHP FTP application or if your `varnish.vcl` file is on an FS Bucket, make sure you redeploy the application for the changes to take effect.</p>
 {{< /alert >}}
 
-To know how to write your `varnish.vcl` file, please have a look at the [Varnish 4.0 book](https://book.varnish-software.com/4.0/).
+To know how to write your `varnish.vcl` file, please have a look at the [Varnish 6 book](https://info.varnish-software.com/resources/varnish-6-by-example-book).
 
-## Varnish 3 to 4 migration
+## Varnish 6 migration
 
-If you already have a configuration file written for varnish 3, you can use [varnish3to4](https://GitHub.com/fgsch/varnish3to4) to convert it to varnish 4 in order to use it on Clever Cloud.
+If you already have a configuration for an older version of varnish, you can read this [guide](https://varnish-cache.org/docs/6.0/whats-new/upgrading-6.0.html) to upgrade to version 6.
```

Comment:
Does it means that varnish will not work anymore for users that have an old configuration file?

Reply:
It depends on the config. But anyway, we don't have any users using varnish 4 (or they have already migrated to version 6)